pkg/injector/patcher/sidecar_container_test.go

```diff
@@ -1019,9 +1179,24 @@ func TestGetSidecarContainer(t *testing.T) {
 	t.Run("sidecar container should have env vars injected", testCaseFn(testCase{
 		annotations: map[string]string{
 			annotations.KeyEnableProfiling: "true",
+			annotations.KeyEnvFromSecret:   "KEY1=NAME1 ,KEY2=NAME2:SECRETKEY2 ",
 		},
 		assertFn: func(t *testing.T, container *corev1.Container) {
 			assert.Contains(t, container.Args, "--enable-profiling")
+			assert.True(t, func() bool {
```

Comment:
This is just a nit, but we seem to be testing to unrelated things in this test, args and env vars...
Seems the name of the test was wrong to begin with?

Reply:
Yes, seems like the test name is a duplicate. Will split and clean up. TY.